aqandu/api_routes.py

```diff
@@ -903,28 +797,41 @@ def getEstimatesForLocations():
         if 'Altitude' not in datum:
             datum['Altitude'] = elevation_interpolator([datum['Longitude']],[datum['Latitude']])[0]
 
+
     # step 7, get estimates from model
-    query_dates = utils.interpolateQueryDates(query_start_datetime, query_end_datetime, query_rate)
-    # note - the elevation grid is the wrong way around, so you need to put in lons first
-    query_elevations = elevation_interpolator(query_lons, query_lats)
-            
-    # step 8, Create Model
-    model, time_offset = gaussian_model_utils.createModel(
-        sensor_data, latlon_length_scale, elevation_length_scale, time_length_scale)
-    # step 9, Compute estimates
-    yPred, yVar = gaussian_model_utils.estimateUsingModel(
-        model, query_lats, query_lons, query_elevations, query_dates, time_offset)
+    # # step 8, Create Model
+    # model, time_offset = gaussian_model_utils.createModel(
+    #     sensor_data, latlon_length_scale, elevation_length_scale, time_length_scale)
+    # # step 9, Compute estimates
+    # yPred, yVar = gaussian_model_utils.estimateUsingModel(
+    #     model, query_lats, query_lons, query_elevations, query_dates, time_offset)
```

Comment:
Can this be removed?

Reply:
This is another case where the code is a work in progress, and until I have more testing done, I want to be able to refer to this.  As this settles we can remove it.  I put a comment above explaining that.